cmd/imagescan/trivy/golang/analyzer/binary/binary.go

```diff
@@ -0,0 +1,48 @@
+package binary
+
+import (
+	"context"
+	"errors"
+	"os"
+
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer/language"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/aquasecurity/trivy/pkg/fanal/utils"
+	"github.com/castai/kvisor/cmd/imagescan/trivy/golang/parser/binary"
+)
+
+func Register() {
```

Comment:
We have separate package for image analyzer. And some examples with custom changes. https://github.com/castai/image-analyzer/blob/main/artifact.go#L32

Would be good to keep in in the same place unless there is a some issue.

Also question.  Does this actually override original trivy Go binary analyzer or it will now run it twice?

Reply:
I thought about adding it to the image-analyzer package, but decide explicitly against it. IMHO we should get rid of the image-analyzer all together, as it only adds complexity, without providing any benefits. 

This will override the original analyizer.